packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsForm/PayableDetailsForm.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import { FC, useState } from 'react';
+
+import { Dialog } from '@/components/Dialog';
+import { action } from '@storybook/addon-actions';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import {
+  PayableDetailsForm,
+  PayableDetailsFormProps,
+} from './PayableDetailsForm';
+
+const meta: Meta<typeof PayableDetailsForm> = {
+  title: 'Payables/Payables — Details Form',
+  component: PayableDetailsForm,
+};
+
+const payableId = '9a3b97a5-a1ba-4d8c-bade-ad3c47ae61e0';
```

Comment:
I would directly use fixture identifier instead of hardcoding. It's hard to maintain.

Reply:
I took this ID from the test cases. Let’s handle it in a separate PR. If you suggest changing the hardcoded values, it’s a good idea to apply this change across the entire project

<img width="706" alt="Screenshot 2024-08-23 at 14 23 18" src="https://github.com/user-attachments/assets/208e85ef-d9bf-42d8-9b9b-3fc88ab3036e">